tokenizers/Cargo.toml

```diff
@@ -72,17 +69,13 @@ esaxx-rs = { version = "0.1.10", default-features = false, features=[]}
 monostate = "0.1.9"
 
 [features]
-default = ["progressbar", "cli", "onig", "esaxx_fast"]
+default = ["progressbar", "onig", "esaxx_fast"]
 esaxx_fast = ["esaxx-rs/cpp"]
 progressbar = ["indicatif"]
 http = ["hf-hub"]
-cli = ["clap"]
 unstable_wasm = ["fancy-regex", "getrandom/js"]
 
 [dev-dependencies]
 criterion = "0.5"
 tempfile = "3.8"
 assert_approx_eq = "1.1"
-
-[profile.release]
```

Comment:
moved to root Cargo.toml since this is where Cargo wants release profiles defined